src/engines/Cesium/common.ts

```diff
@@ -546,15 +546,15 @@ export const colorBlendMode = (colorBlendMode?: "highlight" | "replace" | "mix"
   )[colorBlendMode || ""];
 
 export const colorBlendModeFor3DTile = (
-  colorBlendMode?: "highlight" | "replace" | "mix" | "default",
+  colorBlendMode: "highlight" | "replace" | "mix" | "default" = "default",
 ) =>
   (
     ({
       highlight: Cesium3DTileColorBlendMode.HIGHLIGHT,
       replace: Cesium3DTileColorBlendMode.REPLACE,
       mix: Cesium3DTileColorBlendMode.MIX,
     }) as { [key in string]?: Cesium3DTileColorBlendMode }
-  )[colorBlendMode || ""];
+  )[colorBlendMode || ""] || Cesium3DTileColorBlendMode.HIGHLIGHT;
```

Comment:
And I think you can remove the false case, because `colorBlendMode` variable has default value.
```suggestion
  )[colorBlendMode || ""];
```

Reply:
Currently, there is no default property in the object. In the case `colorBlendMode` parameter is `default` we want the value returned to be `Cesium3DTileColorBlendMode.HIGHLIGHT`

We could remove the logical OR operator and empty string so that we have this:
```suggestion
  )[colorBlendMode] || Cesium3DTileColorBlendMode.HIGHLIGHT;
```

Let me know your thoughts.